README2.md

```diff
@@ -0,0 +1,57 @@
+# Northcoders News API
```

Comment:
Please remove this whole file. The original instructions should not be included in this final branch. You can save the content somewhere on your computer instead if you wanted to.

Reply:
sweet thanks i just deleted it.